media_driver/agnostic/gen11/codec/kernel_free/media_srcs.cmake

```diff
@@ -18,6 +18,35 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
+if (BUILD_KERNELS)
+
+    function(gen_cm_kernel_from_source name platform)
+
+        set(cm_krn_dir ${CMAKE_SOURCE_DIR}/media_driver/agnostic/${platform}/codec/kernel_free)
+        set(cm_krn ig${name}krn_g11)
+
+        # Generating source from the .krn file
+        add_custom_command(
+            OUTPUT ${cm_krn_dir}/${cm_krn}.c ${cm_krn_dir}/${cm_krn}.h
+            DEPENDS KernelBinToSource
+            COMMAND ${CMAKE_COMMAND} -E rename ${CMAKE_CURRENT_LIST_DIR}/commonkernel.krn ${CMAKE_CURRENT_LIST_DIR}/${cm_krn}.krn
+            COMMAND KernelBinToSource ARGS -i ${CMAKE_CURRENT_LIST_DIR}/${cm_krn}.krn -o ${cm_krn_dir} -v ${cm_krn} -index 14 -t 18
+            COMMAND ${CMAKE_COMMAND} -E remove ${CMAKE_CURRENT_LIST_DIR}/${cm_krn}.krn
+            COMMAND ${CMAKE_COMMAND} -E remove_directory ${CMAKE_CURRENT_LIST_DIR}/Binary
+            COMMENT "Generate source file from krn")
+
+    endfunction()
+
+    if(GEN11_ICLLP)
+
+        exec_program(${PYTHON} ${CMAKE_CURRENT_LIST_DIR}
+                    ARGS build.py)
```

Comment:
Configuration fails on my side with:
```
running cd "/home/dvrogozh/git/github/media-driver/media_driver/agnostic/gen11/codec/kernel_free" && /usr/bin/python build.py  2>&1
./Source/downscale_convert_kernel_genx.cpp(23,10):  fatal error: 'cm/cm.h' file not found
#include <cm/cm.h>
         ^~~~~~~~~
1 error generated.
./Source/downscale_kernel_genx.cpp(23,10):  fatal error: 'cm/cm.h' file not found
#include <cm/cm.h>
         ^~~~~~~~~
1 error generated.
./Source/hme_kernel_genx.cpp(23,10):  fatal error: 'cm/cm.h' file not found
#include <cm/cm.h>
         ^~~~~~~~~
1 error generated.
```

I may need to update CM compiler though...